### PR TITLE
VIH-9401 vho dashboard shows hearings allocated to cso

### PR DESCRIPTION
**Pull request by will-craig**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9401

### Change description ###
Updated get conference api response to include the allocated cso; this is displayed in the command-centre

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by shaed-parkar — APPROVED** [review]
Looks good, just a suggestion about the feature flag but I know it's probably got a short shelf life once we go live

File: VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.html [hunk 1]
```diff
@@ -28,7 +28,10 @@
 
   <div class="govuk-grid-row">
     <div class="conference-list" tabindex="0">
-      <app-vho-hearing-list #conferenceList [conferences]="hearings" (selectedConference)="onConferenceSelected($event)">
+      <app-vho-hearing-list #conferenceList
+                            [conferences]="hearings"
+                            [workAllocationFeatureFlag]="vhoWorkAllocationFeatureFlag"
```
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.html:33]: maybe I am wrong but I think that the parent is responsible for the visibility of a child component. This means we can have different visibility logic for the same component. But it is just a preference

[the remark this answers is not in the document]
Comment (will-craig) [reply VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.html:33]: That was my thinking ^

File: VideoWeb/VideoWeb/Controllers/ConferencesController.cs [hunk 2]
```diff
@@ -168,6 +176,21 @@ public async Task<ActionResult<List<ConferenceForVhOfficerResponse>>> GetConfere
...
+            foreach (var hearings in allocatedHearings)
+            {
+                var conference = responses.FirstOrDefault(conference => hearings.HearingId == conference.HearingRefId);
+                if (conference == null)
+                {
+                    _logger.LogWarning("Allocated hearing id, not in list of conferences for response");
+                    continue;
+                }
+
+                conference.AllocatedCso = hearings?.Cso?.FullName ?? "Unallocated";
```
Comment (marcogagliardi) [inline VideoWeb/VideoWeb/Controllers/ConferencesController.cs:190]: I think here we should have a logic for the labelling. There is a logic for displaying the right label in the hearing mapper.

**Pull request merged**